service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationProcessors.java

```diff
@@ -86,7 +88,8 @@ private static GeneratePdfPayload getGeneratePdfPayload(MasProcessingObject tran
     MasAutomatedClaimPayload claimPayload = transferObject.getClaimPayload();
     GeneratePdfPayload generatePdfPayload = new GeneratePdfPayload();
     generatePdfPayload.setEvidence(transferObject.getEvidence());
-    generatePdfPayload.setClaimSubmissionId(claimPayload.getClaimDetail().getBenefitClaimId());
+    generatePdfPayload.setClaimSubmissionId(String.valueOf(claimPayload.getCollectionId()));
```

Comment:
Should be a String:
```suggestion
    generatePdfPayload.setClaimSubmissionId(claimPayload.getCollectionId());
```

Reply:
CollectionId is an integer in the mas payload so it is not a string here. At this point we are out of time I think to make more changes throughout the code for this PR to make it a string earlier. At least without serious risk to the deadline there is another PR stacked behind this one with more schema changes.